diff --git a/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.lib b/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.lib
index 966cb19..f82be5c 100644
Binary files a/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.lib and b/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.lib differ
diff --git a/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.vcproj b/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.vcproj
index 69114c8..1b7b5eb 100644
--- a/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.vcproj
+++ b/Coding/matrix-file-reader/igor-xop/VC8/XOP_Toolkit_6/IgorXOPs6/XOPSupport/VC8/XOPSupport.vcproj
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="Windows-1252"?>
 <VisualStudioProject
 	ProjectType="Visual C++"
-	Version="8.00"
+	Version="8,00"
 	Name="XOPSupport"
 	ProjectGUID="{40FBCE38-EF59-469E-8C7C-5D672A0CAFFC}"
 	>
@@ -21,7 +21,7 @@
 			OutputDirectory="$(SolutionDir)$(ConfigurationName)"
 			IntermediateDirectory="$(ConfigurationName)"
 			ConfigurationType="4"
-			UseOfMFC="0"
+			UseOfMFC="2"
 			ATLMinimizesCRunTimeLibraryUsage="false"
 			CharacterSet="2"
 			>
@@ -46,7 +46,7 @@
 				AdditionalIncludeDirectories=""
 				PreprocessorDefinitions="WIN32;_WIN32_WINNT=0x0500;WINVER=0x0500;NDEBUG;_LIB;_CRT_SECURE_NO_WARNINGS"
 				StringPooling="true"
-				RuntimeLibrary="0"
+				RuntimeLibrary="2"
 				StructMemberAlignment="0"
 				EnableFunctionLevelLinking="false"
 				UsePrecompiledHeader="0"
